core/src/main/scala/kafka/cluster/Partition.scala

```diff
@@ -578,7 +579,8 @@ class Partition(val topicPartition: TopicPartition,
       // Updating the assignment and ISR state is safe if the partition epoch is
       // larger or equal to the current partition epoch.
       updateAssignmentAndIsr(
-        assignment = partitionState.replicas.asScala.map(_.toInt),
+        assignment = replicas,
```

Comment:
Would it make sense to change `assignment` to `replicas`?

Reply:
That makes sense.